lib/graphql/subscriptions/event.rb

```diff
@@ -37,7 +37,7 @@ def self.serialize(name, arguments, field, scope:)
           arguments
         when Hash
           if field.is_a?(GraphQL::Schema::Field)
-            stringify_args(arguments)
+            stringify_args(arguments, field)
```

Comment:
The definition below has `arg_owner, args` for the parameters, should these be switched?

```suggestion
            stringify_args(field, arguments)
```

Reply:
Ou, yes it should.

Reply:
So, I have updated the tests and add small fix for stringify_args method with auto-loaded arguments, I am not sure, if it is the best solution, so I am open for a next discusion and suggestions.